website/docs/index.html.markdown

```diff
@@ -58,7 +59,8 @@ The following keys can be used to configure the provider.
   details. On your computer, if you have made your identity available as the
   Application Default Credentials by running [`gcloud auth application-default
   login`](https://cloud.google.com/sdk/gcloud/reference/auth/application-default/login),
-  the provider will use your identity.
+  the provider will use your identity. However, this method is not guaranteed to
```

Comment:
Consider putting it in bold or as a `Note`. I think this is a very important caveat.

Reply:
Not sure if you know this, but if you put `~>` and then text like:

```
~> This is a note
```

It'll get called out with the right CSS to actually look like a note. There's also:

- `!>` for big red warnings
- `->` for info
- `=>` for green success

I would like to link to the `service accounts` section or some kind of documentation. We say "this is the easiest way, but it might break in edge cases", name the replacement, but then don't provide a way to get more information on the replacement.

Reply:
Yeah, by `Note`, I meant `~>` like we do [here](https://www.terraform.io/docs/providers/google/r/compute_disk.html).

But I didn't know about `=>` and `->`. Thanks for sharing this tip :)